src/main/kotlin/solve/settings/visualization/popover/SettingsDialogNode.kt

```diff
@@ -0,0 +1,19 @@
+package solve.settings.visualization.popover
+
+import javafx.scene.layout.VBox
+import solve.styles.Style
+import tornadofx.*
+
+class SettingsDialogNode : VBox(10.0) {
+
+    var title: String = ""
```

Comment:
public field? Why it is mutable?

Reply:
field is public because its value is set in another class, when possible define title